core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -413,6 +408,12 @@ class CivilizationInfo {
         else
             otherCiv.addNotification(meetString, NotificationIcon.Gold)
 
+        if (otherCiv.isCityState() && otherCiv.cityStateType == CityStateType.Religious){
```

Comment:
otherCiv.canGiveStat(Stat.Faith) would be more robust

Reply:
fixed

Reply:
fixed